src/element/textWysiwyg.tsx

```diff
@@ -117,14 +120,25 @@ export const textWysiwyg = ({
     if (updatedTextElement && isTextElement(updatedTextElement)) {
       const coordX = updatedTextElement.x;
       let coordY = updatedTextElement.y;
+      let eCoordY = coordY;
+      let rCoordY = coordY;
       const container = getContainerElement(updatedTextElement);
-      let maxWidth = updatedTextElement.width;
 
-      let maxHeight = updatedTextElement.height;
-      const width = updatedTextElement.width;
+      // Editing metrics
+      const eMetrics = measureText(
```

Comment:
Haven't tested the perf but I think calling this function on every keystroke would turn out to be very expensive.
Right now in prod its called once per keystroke and in this PR its called thrice which will significantly affect the perf

Also, we should avoid calling this function here since we are trying the compute the position of text in the container which could be accurately determined from container dimensions.

Reply:
I have updated this PR to address the perf issues you mentioned.